### ISSUE TITLE
dotfiles task fails on bootstrap due to git setup

### ISSUE THREAD

**Issue by Flydiverny** [caarlos0/machine#9]
Hi! Like what you are doing here, been checking out a few dotfiles repos lately and decided to test yours out. This looks like a nice repo to setup a new machine but I ran into a few kinks :)

Currently it fails in the dotfiles setup if you haven't setup git beforehand, due to the prompts.

```
TASK [Setup dotfiles] **********************************************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": "cd ~/.dotfiles && ./script/bootstrap", "delta": "0:00:00.013797", "end": "2016-07-13 10:28:18.463835", "failed": true, "rc": 1, "start": "2016-07-13 10:28:18.450038", "stderr": "", "stdout": "\n\r  [ \u001b[00;34m..\u001b[0m ] setup gitconfig\n\r  [ \u001b[0;33m??\u001b[0m ]  - What is your github author name?", "stdout_lines": ["", "", "  [ \u001b[00;34m..\u001b[0m ] setup gitconfig", "", "  [ \u001b[0;33m??\u001b[0m ]  - What is your github author name?"], "warnings": []}
    to retry, use: --limit @main.retry
```

**Comment by Flydiverny** [comment]
If setting up gitconfig ahead of time the machine setup also wont finish due to the dotfiles bootstrapper killing the terminal when its done. Maybe the dotfiles task should be last?

**Comment by caarlos0** [comment]
hmmm, good catch, I'll try to work this out... thanks

**Comment by caarlos0** [comment]
@Flydiverny I'm testing a fix in #10 👍
